1.16 forge

Stereowalker opened the pull request:
Well, here it is... The official forge version

Stereowalker committed:
Initial Forge Port

Stereowalker committed:
Updated mod version

Stereowalker committed:
Update README.md

tyra314 commented on src/main/java/hunternif/mc/impl/atlas/ClientProxy.java:
```diff
@@ -41,37 +131,32 @@ public void initClient() {
 		textureMap = BiomeTextureMap.instance();
 		registerVanillaCustomTileTextures();
-
+		IReloadableResourceManager r;
```
This doesn't seem to be used.

tyra314 commented on src/main/java/hunternif/mc/impl/atlas/ClientProxy.java:
```diff
@@ -41,37 +131,32 @@ public void initClient() {
 //		MarkerType.REGISTRY.setDirty(true);
-
 		for (MarkerType type : MarkerType.REGISTRY) {
+			System.out.println("Amen to "+type.getIcon());
```
Debug printf

tyra314 commented on src/main/resources/META-INF/coremods - Copy.json:
```diff
@@ -0,0 +1,5 @@
+
```
this file seems not necessary

tyra314 commented on src/main/resources/antiqueatlas.mixins.json:
```diff
@@ -2,6 +2,7 @@
   "required": true,
   "package": "hunternif.mc.impl.atlas.mixin",
   "compatibilityLevel": "JAVA_8",
+  "refmap": "sereneseasons.refmap.json",
```
this looks like a copy-pasta-error?

tyra314 commented on src/main/resources/antiqueatlasoverlay.mixins.json:
```diff
@@ -2,6 +2,7 @@
   "required": true,
   "package": "kenkron.antiqueatlasoverlay.mixin",
   "compatibilityLevel": "JAVA_8",
+  "refmap": "sereneseasons.refmap.json",
```
```suggestion
  "refmap": "antiqueatlas.refmap.json",
```

tyra314 commented on src/main/resources/antiqueatlasoverlay.refmap.json:
```diff
@@ -0,0 +1,16 @@
+  "required": true,
+  "package": "kenkron.antiqueatlasoverlay.mixin",
+  "compatibilityLevel": "JAVA_8",
+  "refmap": "sereneseasons.refmap.json",
```
```suggestion
  "refmap": "sereneseasons.refmap.json",
```

tyra314 reviewed (changes requested):
Some small things.

Also, I'd prefer if you could integrate the classes under the `stereowalker` package into someplace under `hunternif.antiqueatlas`, wherever they fit.

tyra314 commented:
Also, please add the github actions file from here: https://github.com/AntiqueAtlasTeam/AntiqueAtlas/blob/1.16-fabric/.github/workflows/gradle.yml (and add their setupCIWorkspace call needed for forge)

I can't be bothered with Travis anymore.

Stereowalker committed:
Forge Port Update

Removed mixins as they were already being handled by coremods, moved the files in the `stereowalker.forge` package to 'hunternif.mc.impl.atlas.forge' and other changes

Stereowalker committed:
Used a custom 'ItemCraftedEvent' rather than forge's 'ItemCraftedEvent'

Stereowalker committed:
Update transformer.js

Stereowalker committed:
Update mods.toml

Included all contributors in the credits

tyra314 commented on src/main/java/hunternif/mc/impl/atlas/api/oldimpl/TileApiImpl.java:
```diff
@@ -51,13 +51,17 @@
 	
 	public TileApiImpl() {
-		TileIdRegisteredCallback.EVENT.register(this::onTileIdRegistered);
+		//FIXME TileIdRegisteredCallback.EVENT.register(this::onTileIdRegistered);
```
Looking at the next method, this FIXME seems done.

tyra314 commented on src/main/java/hunternif/mc/impl/atlas/network/packet/s2c/play/TileNameS2CPacket.java:
```diff
@@ -1,20 +1,22 @@
 import hunternif.mc.impl.atlas.ext.ExtTileTextureMap;
-import hunternif.mc.impl.atlas.ext.TileIdRegisteredCallback;
+import hunternif.mc.impl.atlas.forge.hook.AntiqueAtlasHooks;
+//import hunternif.mc.impl.atlas.ext.TileIdRegisteredCallback;
```
Please remove the unused import

tyra314 commented on src/main/java/hunternif/mc/impl/atlas/network/packet/s2c/play/TileNameS2CPacket.java:
```diff
@@ -23,41 +25,69 @@
+			BiomeTextureMap.instance().setTexture(id, texture);
+		}
+
+		//FIXME TileIdRegisteredCallback.EVENT.invoker().onTileIDsReceived(tileIds);
```
Seems fixed

tyra314 reviewed (changes requested):
Looks good, though three more small things.

Stereowalker committed:
Applied Required Changes

Stereowalker committed:
Update transformer.js
